packages/dds/tree/src/test/edit-manager/editManager.spec.ts

```diff
@@ -246,17 +253,18 @@ describe("EditManager", () => {
             refNumber: brand(2),
             changeset: TestChangeRebaser.mintChangeset([1, 2], 3),
         };
-        manager.addLocalChange(c1.changeset);
-        manager.addSequencedChange(c1);
-        manager.addLocalChange(c2.changeset);
-        manager.addSequencedChange(c2);
-        manager.addLocalChange(c3.changeset);
-        manager.addSequencedChange(c3);
+        assert.deepEqual(manager.addLocalChange(c1.changeset), asDelta([1]));
```

Comment:
It might be nice to have helpers like

```
addLocalChangeAndCheckDelta(manager, c1.changeset);
addSequencedChangeAndCheckDelta(manager, c1.changeset);
```

The number of lines affected by the recent changes to the test design makes me feel like trying to do a larger refactoring to deduplicate logic in these tests and make them more succinct.

Reply:
How do you know what the delta is supposed to be in the `addSequencedChangeAndCheckDelta` case?

Reply:
When adding a local change we always expect the direct delta from the change. For sequenced changes it's probably not worth changing, but we always expect an empty delta if the commit's session ID is the local ID, and in the vast majority of the other calls we expect the delta to be the direct delta from the change. If we specified these tests in a format more like the generated scenarios, then we could calculate the delta like we do there.

Reply:
As discussed, I will have a go at refactoring that in a separate PR.